src/biogeophys/SectorWaterMod.F90

```diff
@@ -0,0 +1,1238 @@
+          
+          !
+          ! !LOCAL VARIABLES:
+          integer :: g       ! gridcell index
+          integer :: year    ! year (0, ...) for nstep+1
+          integer :: mon     ! month (1, ..., 12) for nstep+1
+          integer :: day     ! day of month (1, ..., 31) for nstep+1
+          integer :: sec     ! seconds into current date for nstep+1
+
+          integer :: first_read ! variable to do first read, in future I may prefer to make a subroutine is_beg_curr_month to avoid exception for first reading
+
+          real(r8) :: dayspyr                                    ! days per year
+          real(r8) :: dayspm                                     ! days per month
+          real(r8) :: secs_per_day                               ! seconds per day
+          real(r8) :: flux_transform_from_monthly_to_second      ! factor to transform the demand from mm/month to mm/s for the given day
+
+          real(r8) :: dom_demand(bounds%begg:bounds%endg)
+          real(r8) :: dom_demand_volr_limited(bounds%begg:bounds%endg)
+          
+          real(r8) :: dom_consumption(bounds%begg:bounds%endg)
+          real(r8) :: dom_consumption_volr_limited(bounds%begg:bounds%endg)
+          
+          real(r8) :: liv_demand(bounds%begg:bounds%endg)
+          real(r8) :: liv_demand_volr_limited(bounds%begg:bounds%endg)
+          
+          real(r8) :: liv_consumption(bounds%begg:bounds%endg)
+          real(r8) :: liv_consumption_volr_limited(bounds%begg:bounds%endg)
+          
+          real(r8) :: elec_demand(bounds%begg:bounds%endg)
+          real(r8) :: elec_demand_volr_limited(bounds%begg:bounds%endg)
+          
+          real(r8) :: elec_consumption(bounds%begg:bounds%endg)
+          real(r8) :: elec_consumption_volr_limited(bounds%begg:bounds%endg)
+          
+          real(r8) :: mfc_demand(bounds%begg:bounds%endg)
+          real(r8) :: mfc_demand_volr_limited(bounds%begg:bounds%endg)
+          
+          real(r8) :: mfc_consumption(bounds%begg:bounds%endg)
+          real(r8) :: mfc_consumption_volr_limited(bounds%begg:bounds%endg)
+          
+          real(r8) :: min_demand(bounds%begg:bounds%endg)
+          real(r8) :: min_demand_volr_limited(bounds%begg:bounds%endg)
+          
+          real(r8) :: min_consumption(bounds%begg:bounds%endg)
+          real(r8) :: min_consumption_volr_limited(bounds%begg:bounds%endg)
+          
+          ! Whether we should limit deficits by available volr
+          logical :: limit_sectorwater
+          
+          character(len=*), parameter :: subname = 'CalcSectorWaterNeeded'
+          !-----------------------------------------------------------------------
+ 
+          ! Get current date
+          call get_curr_date(year, mon, day, sec)
+          dayspyr = get_curr_days_per_year()
+          dayspm  = dayspyr/12._r8
+          first_read = 1
+          
+          flux_transform_from_monthly_to_second = (1._r8/dayspm)/86400._r8
```

Comment:
The 86400 here and elsewhere in this file should use the secspday parameter from clm_varcon.

Reply:
Agreed! This is something I have noted in my "minor fixes I'll do" that I didn't bother putting in my review.